### ISSUE TITLE
Properly update contact birthdays at change

### ISSUE THREAD

**Issue by tibbi** [SimpleMobileTools/Simple-Calendar#938]

**Comment by aunwin** [comment]
Not sure if you prefer a new issue or if this belongs to properly update:If one includes contact birthdays via the app multiple times because e.g. a new contact with birthday has become part of the phone book but is not yet included in the calendar, all already existing contacts exist twice then, blocking the space to see other appointments during those days. It would be great if updating contact birthdays would also deduplicate or prevent duplication in the first place.

**Comment by tibbi** [comment]
there should be duplication avoiding already, it works on my devices. What app version do you have?

**Comment by aunwin** [comment]
I use version 6.7.2 Pro.
After your message I investigated a little deeper and it turns out the duplication happens because Simple Calender creates 'only local' entries for the imported birthdays and I have manually changed the first import to be part of a synced calendar that happened to have the same colour. Therefore after reimporting the birthdays showed up twice but were only once in the local calendar. So sorry for the confusion.

**Comment by pcsaunak** [comment]
I did change the birthday of a user and saw that it got updated correctly.
